std_msgs/Header header
string tracker_name
MocapMarker[] markers
